builtin/providers/aws/resource_aws_sns_topic_subscription.go

```diff
@@ -44,6 +45,24 @@ func resourceAwsSnsTopicSubscription() *schema.Resource {
 				Required: true,
 				ForceNew: false,
 			},
+			"endpoint_auto_confirms": &schema.Schema{
+				Type:     schema.TypeBool,
+				Optional: true,
+				ForceNew: false,
```

Comment:
I see your point but almost all the schema elements have ForceNew: false, I guess they made a bad copy :). I will fix it.